operator/src/main/java/oracle/kubernetes/operator/helpers/DomainPresenceInfo.java

```diff
@@ -429,7 +429,7 @@ public int incrementAndGetFailureCount() {
     return retryCount.incrementAndGet();
   }
 
-  int getRetryCount() {
+  public int getRetryCount() {
```

Comment:
It looks like this is only called from getActiveDeadlineSeconds() in JobStepContext in same package and can be package protected instead of public.